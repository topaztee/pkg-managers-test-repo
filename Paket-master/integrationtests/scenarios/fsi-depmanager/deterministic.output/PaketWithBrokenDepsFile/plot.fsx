

#r "paket: "

open XPlot.Plotly

Chart.Line [ 1 .. 10 ]