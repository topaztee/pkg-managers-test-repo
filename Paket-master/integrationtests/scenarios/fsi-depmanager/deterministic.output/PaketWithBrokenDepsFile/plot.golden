

plot.fsx(5,6): error FS0039: The namespace or module 'XPlot' is not defined.

